### PR TITLE
Extra conformance tests for `verifyEcdsaSecp256k1Signature`

### PR DESCRIPTION
**Pull request by kwxm**
This follows up on #6617 by adding extra conformance tests for `verifyEcdsaSecp256k1Signature`.  I couldn't find any official test vectors so I generated some myself using `openssl`.  That was quite complicated so I've included a README describing the procedure in case we ever need to do it again.

This should close issue #6553.

### CODE REVIEW

File: plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/verifyEcdsaSecp256k1Signature/test-vector-16/test-vector-16.uplc.budget.expected [hunk 1]
```diff
@@ -0,0 +1 @@
+evaluation failure
```
Comment (effectfully) [inline plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/verifyEcdsaSecp256k1Signature/test-vector-16/test-vector-16.uplc.budget.expected:1]: Well, that's not exactly a budget.

File: plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/verifyEcdsaSecp256k1Signature/test-vector-17/test-vector-17.uplc.budget.expected [hunk 2]
```diff
@@ -0,0 +1 @@
+evaluation failure
```
Comment (effectfully) [inline plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/verifyEcdsaSecp256k1Signature/test-vector-17/test-vector-17.uplc.budget.expected:1]: Same.

File: plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/verifyEcdsaSecp256k1Signature/test-vector-18/test-vector-18.uplc.budget.expected [hunk 3]
```diff
@@ -0,0 +1 @@
+evaluation failure
```
Comment (effectfully) [inline plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/verifyEcdsaSecp256k1Signature/test-vector-18/test-vector-18.uplc.budget.expected:1]: Same.

File: plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/verifyEcdsaSecp256k1Signature/README.md
Comment (effectfully) [inline plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/verifyEcdsaSecp256k1Signature/README.md:1]: Amazing docs.

**Review by effectfully — APPROVED** [review]
That's great work.

File: plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/verifyEcdsaSecp256k1Signature/test-vector-16/test-vector-16.uplc.budget.expected (hunk 1 above)
Comment (kwxm) [reply plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/verifyEcdsaSecp256k1Signature/test-vector-16/test-vector-16.uplc.budget.expected:1]: It's like that for all of the  tests that produce `error` ([here](https://github.com/IntersectMBO/plutus/blob/master/plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/divideInteger/divideInteger-zero/divideInteger-zero.uplc.budget.expected) for instance).  Maybe we should fix it?

**Pull request merged**